src/view/com/composer/SelectMediaBtn.tsx

```diff
@@ -0,0 +1,329 @@
+import {atoms as a, useTheme} from '#/alf'
+import {Button} from '#/components/Button'
+import {ComposerImage, createComposerImage} from '#/state/gallery'
+import {getDataUriSize} from '#/lib/media/util'
+import {getVideoMetadata} from '#/view/com/composer/videos/pickVideo'
+import {Image_Stroke2_Corner0_Rounded as Image} from '#/components/icons/Image'
+import {isNative, isWeb} from '#/platform/detection'
+import {msg} from '@lingui/macro'
```

Comment:
```suggestion
import {msg, plural} from '@lingui/macro'
```
Would it be possible to add plural formatting please so that the new toast strings can be correctly localized? 🙏